src/main/java/no/sikt/sws/PrefixStripper.java

```diff
@@ -31,11 +33,11 @@ public static JsonNode node(JsonNode node, String workspacePrefix) {
             return null;
         }
 
-        var regex = "(?<=[ /\"\\[])" + workspacePrefix + "-";
+        var regex = "([ /\"\\[])(" + workspacePrefix + "-)(.*?[\"/ ])";
 
         return string2JsonNode(
             node.toString()
-                .replaceAll(regex, EMPTY_STRING));
+                .replaceAll(regex, "$1$3"));
```

Comment:
whats this? If you extract is as a final variable you can give a hint in its name

Reply:
fixed